src/pages/index.tsx

```diff
@@ -5,73 +5,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import { Badges, Contributors } from '../components';
+
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import React from 'react';
 import classnames from 'classnames';
-import styles from './styles.module.css';
+import styles from '../css/styles.module.css';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 
```

Comment:
Nope please do not change our main page it is designed our side.
Just applied in https://website.hackatalk.dev